modal/README.md

```diff
@@ -0,0 +1,59 @@
+# Modal
+
+## Overview
+
+[Modal][1] lets you run generative AI models, large-scale batch jobs, job
+queues, and much more. The Datadog integration collects logs and metrics from
+your applications for you to observe directly in your Datadog account.
+
+## Setup
+
+### Installation
+
+To set up the Modal integration:
+
+1. On the [Datadog Integrations page][2], search for the **Modal** tile and
+   click **Install Integration**.
+
+2. Click **Connect Accounts** to begin authorization of this integration. You
+   will be redirected to log into [Modal][1], and once logged in, you'll be
+   redirected to the Datadog authorization page.
+
+3. Click the **Authorize** button to complete setup.
+
+Logs and metrics from your Modal apps should now start appearing in Datadog.
+
+## Data Collected
+
+### Metrics
+
+Modal collects utilization metrics for applications/functions that run on Modal.
+You can view these metrics on your [Modal][1] dashboard.
```

Comment:
It may be more helpful to mention where users can find these metrics and other data within Datadog, rather than within Modal.